sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -98,19 +92,16 @@ public UserDto selectActiveUserByLogin(DbSession session, String login) {
    * if list of logins is empty, without any db round trips.
    */
   public List<UserDto> selectByLogins(DbSession session, Collection<String> logins) {
-    return executeLargeInputs(logins, session.getMapper(UserMapper.class)::selectByLogins);
+    return executeLargeInputs(logins, getMapper(session)::selectByLogins);
   }
 
   /**
    * @deprecated since 6.0 please use {@link #selectByLogins(DbSession, Collection)} instead
    */
   @Deprecated
   public List<UserDto> selectByLogins(Collection<String> logins) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)